src/lib/routes/admin-api/project/variants.ts

```diff
@@ -65,6 +65,9 @@ export default class VariantsController extends Controller {
             handler: this.getVariants,
             middleware: [
                 openApiService.validPath({
+                    summary:
```

Comment:
This is too long to be a summary and should be a description instead. Let's try and add a summary at the same time.
```suggestion
                    summary: 'Retrieve variants for a feature (deprecated) ',
                    description:
```